### ISSUE TITLE
connect router to zigbee2mqtt

### ISSUE THREAD

**Issue by pvklink** [seaky/nspanel_pro_tools_apk#160]
Hi, i have a nspanel 86p with version 2.3.1. firmware 1.6.0
I try to connect this device as a zigbee router to my working zigbee2mqtt docker instance on a rpi 4b.
So i enabled the pairing mode on my zigbee2mqtt and
on the nspanel, 
enabled zigbee
enabled router mode
enabled pairing mode

but no connection!

Do i need extra software for zigbee on the nspanel or enable some software?
my nspanel gives zigbee state online

**Comment by seaky** [comment]
Try to restart the device and after pair!

**Comment by pvklink** [comment]
nope, no entries or a sign of life in zigbee2mqtt Docker, no discovering of the panel
adding a normal switch to my zigbee2mqtt works fine!

**Comment by seaky** [comment]
With z2m 1.41 it detects automatically and identifes as NSPanel Pro Router.

Oooohhhhhh btw what is your FW version?

**Comment by seaky** [comment]
I see you are on 1.6.0 so it wont work definitelly if you want to test I can provide you a beta with custom zigbee stack installation

**Comment by pvklink** [comment]
like to have the beta custom zigbee stack installation!
I assume i have to put the zigbee function to OFF in my two panels , both 1.6 as you advised...

**Comment by seaky** [comment]
yes under 1.10 much wider customizations are available. 
From fw release to release they are locking down the OS more and more. For example as of version 3.0 u can no longer adjust the proximity sensor, and since version 1.10 you cant acces touch functions.  They modified it on kernel level so unable to bypass.

The only reason to check for updates is that the Zigbee stack is continously updated, which is crucial for recognizing new Sonoff devices.

That is the reason why I planned to add the ability to change that part of the OS.

**Comment by seaky** [comment]
btw the firmware on EFR Zigbee chip have not changed since the first production thats why I assumed the firmware on the chip is fully functional. I also checked it and it looks the fw on chip is stock Siliconlabs emberznet 6.7

**Comment by pvklink** [comment]
For me its easy, just follow your advises and use your software and very happy...

**Comment by Foguet** [comment]
I have the firmware 1.7.0 if I can help you to test new version, I'm here (obviously I can't link my nspanel to my zigbee2mqtt either)

**Comment by pvklink** [comment]
same for me, lots of free days coming month, so when you are ready to create the version, give us a sign

**Comment by seaky** [comment]
beta will be available soon...

**Comment by pvklink** [comment]
can't wait :-)

**Comment by pvklink** [comment]
by the way , 2.3.1 works great for me!
Just used the functionality to set the brightness of the screen at night, when there is no light in combination with the sleep settings, works great!

**Comment by seaky** [comment]
https://github.com/seaky/nspanel_pro_tools_apk/discussions/164#discussion-7574260

**Comment by seaky** [comment]
> does this beta works for nspanel with firmware 1.7 ?

even on 1.3 :) I have some devices with 1.3 and works well. 
btw it is harmless because the installer just replaces some binaries what you can recover anytime. It does not touch chip firmesres such as the zigbee chip fw.

**Comment by pvklink** [comment]
Ok , i am a little bit nervous to install the beta, it works so well!
But i am curious about the possiblities!
Can i use my nspanel 
- as a router that will be recognized by/added to zigbee2mqtt(docker)
- as a coordinator for zigbee2mqtt(docker) , it needs an ip and a port...

**Comment by seaky** [comment]
beta3 release with z2m support

**Comment by Helvio88** [comment]
Running Beta 3, installed on top of Beta 2, zero issues 🎯
